gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/sql/execute/SnappyUpdateDeleteResultSet.java

```diff
@@ -40,7 +40,7 @@ public int modifiedRowCount() {
       rowCount = 0;
       try {
         ExecRow row;
-        while ((row = getNextRow()) != null) {
+        while ((row = getNextRow()) != null && row.nColumns() > 0) {
```

Comment:
@sumwale, as shown in
https://jira.snappydata.io/browse/SNAP-1451?focusedCommentId=17172&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17172

and

https://jira.snappydata.io/browse/SNAP-1451?focusedCommentId=17173&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17173,

the return of snc.sql itself return no rows. I think that needs to be corrected first.